scripts/zdev-generate.py

```diff
@@ -12,6 +12,6 @@
     record = dict([k.split('=',1) for k in row])
     if record['COLUMN'] not in ('TYPE', 'ATTR:', 'ATTRPATH:'):
         desired_columns.append(record['COLUMN'].lower())
-cmd = ['lszdev','--pairs','--columns',','.join(desired_columns)]
+cmd = ['lszdev', '--quiet', '--pairs','--columns',','.join(desired_columns)]
```

Comment:
Thanks for looking into this!
If I can nitpick, it would be nice to be consistent on the spaces.  The rest of the array doesn't use them.  Go with a space after the comma for the rest of the array please, that way it's a little closer to passing the linter.

Reply:
ok. I adjusted the whitespaces
(I c&p-ed from the other file, that had the whitespaces)